### PR TITLE
Allow to toggle off getting files with CDN url

### PR DESCRIPTION
**Pull request by hcarneiro**

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ var data = Fliplet.Widget.getData() || {};
 data.type = data.type || '';
 data.selectFiles = data.selectFiles || [];
 data.autoSelectOnUpload = data.autoSelectOnUpload || false;
+data.cdnToggle = data.cdn || true;
```
Comment (squallstar) [inline js/interface.js]: Please make it so when it's not sent it will not send a specific value.

Comment (squallstar) [reply js/interface.js]: @hcarneiro also, why calling it toggle? just keep it called as `cdn` or `useCdn`

File: js/interface.js [hunk 2]
```diff
@@ -455,7 +456,8 @@ function getFolderAndFiles(filter) {
 
   return Promise.all([
     Fliplet.Media.Folders.get($.extend({}, {
-      type: 'folders'
+      type: 'folders',
+      cdn: data.cdnToggle
```
Comment (squallstar) [inline js/interface.js]: you could just pass `data.cdn` and you're good to go

Comment (hcarneiro) [reply js/interface.js]: what if it is `undefined` ?

Comment (squallstar) [reply js/interface.js]: @hcarneiro the backend has logic to automatically deal with what the default option should be when not given

**Pull request merged**